builders/integrations/oracles/chainlink.md

```diff
@@ -9,197 +9,110 @@ description: How to use request data from a Chainlink Oracle in your Moonbeam Et
-
-### Run your Client Contract {: #run-your-client-contract } 
-
-If you want to run your Client contract but use our Oracle node, you can do so with the following information:
-
-|  Contract Type  |     |                      Address                      |
-| :-------------: | --- | :-----------------------------------------------: |
-| Oracle Contract |     | {{ networks.moonbase.chainlink.oracle_contract }} |
-|   LINK Token    |     |  {{ networks.moonbase.chainlink.link_contract }}  |
-
-Remember that the LINK token payment is set to zero.
-
-### Other Requests {: #other-requests } 
-
-Chainlink's Oracles can tentatively provide many different types of data feeds with the use of external adapters. However, for simplicity, our Oracle node is configured to deliver only price feeds.
-
-If you are interested in running your own Oracle node in Moonbeam, please visit [this guide](/node-operators/oracle-nodes/node-chainlink/). Also, we recommend going through [Chainlink's documentation site](https://docs.chain.link/docs).
+Developers can now use [Chainlink's decentralized Oracle network](https://chain.link/) to fetch data in the Moonbase Alpha TestNet and Moonriver. [Price Feeds](https://docs.chain.link/docs/architecture-decentralized-model) contain real-time price data that is continuously updated by Oracle operators in a smart contract so that other smart contracts can fetch and consume it. This guide will cover the available price feeds and how to fetch the latest price data on Moonriver. 
 
 ## Price Feeds {: #price-feeds } 
 
 Before we go into fetching the data itself, it is important to understand the basics of price feeds.
 
-In a standard configuration, each price feed is updated by a decentralized Oracle network. Each Oracle node is rewarded for publishing the price data to the Aggregator contract. However, the information is only updated if a minimum number of responses from Oracle nodes are received (during an aggregation round).
+In a standard configuration, each price feed is updated by a decentralized Oracle network. Each Oracle node is rewarded for publishing the price data to the Aggregator contract. The Aggregator contract receives periodic data updates from the network of oracles and aggregates and stores the data on-chain so that consumers can easily fetch it. However, the information is only updated if a minimum number of responses from Oracle nodes are received (during an aggregation round).
 
-The end-user can retrieve price feeds with read-only operations via a Consumer contract, referencing the correct Aggregator interface (Proxy contract). The Proxy acts as a middleware to provide the Consumer with the most up-to-date Aggregator for a particular price feed.
+The end-user can retrieve price feeds with read-only operations via an Aggregator interface.
 
-![Price Feed Diagram](/images/builders/integrations/oracles/chainlink/chainlink-price-feed.png)
```

Comment:
Why did you remove this image :( haha I kind of like to have an image so it is not all text

Reply:
I removed it because we don't talk about the consumer contract anymore so I thought it would just add more confusion

Reply:
I get what you're saying though, I did add 1 more screenshot to this page though if that counts for anything hahah

Reply:
Hahaha. But I think the consumer contract can still be used. I got this diagram from Chainlink official docs

Reply:
ok I'll add it back in!